lib/test/tasks/create_db_copies.rb

```diff
@@ -32,7 +35,7 @@ def run
       end
 
       execute_system_command(<<~COMMAND.squish)
-        psql -U postgres -h 127.0.0.1 -d david_runger_test -c "
+        psql -U #{postgres_user} -h #{postgres_host} -d david_runger_test -c "
```

Comment:
This should have been done in #6306 . Since that PR, I believe that this step has been failing locally. I didn't run this step locally until now.